### ISSUE TITLE
Flatten newline into <br>

### ISSUE THREAD

**Issue by revilwang** [Mermade/widdershins#211]
**Is your feature request related to a problem? Please describe.**

When transform OpenAPI3 specs, the newline `'\n'` characters in long description are flattened to blank spaces, which leads to description cell really hard to read.

**Describe the solution you'd like**

I would suggest that blank space is flattened into `<br>` so that the original format is preserved.

**Describe alternatives you've considered**

Or provides a configuration to allow so

**Comment by MikeRalphson** [comment]
Interesting. Can you show a fragment of your OpenAPI document so I can see how the `\n` characters appear, i.e. to make sure the line-breaks are not just YAML formatting etc?

Then I'll have a play and see how things look.

**Comment by revilwang** [comment]
OpenAPI specs says the description can be written as CommonMark format, and we utilize this function a lot. One of my schema component has a field defined as below:

```yaml
status:
  type: string
  readOnly: true
  enum:
    - open
    - close
  description: |
      Order status

      - `open`: waiting to be traded
      - `finished`: finished
```

But in the Markdown file widdershins generated is shown as below:

```
|status|string|false|read-only|Order status  - `open`: waiting to be traded - `finished`: finished|
```

The newline characters are flattened to blank space. If I manually edit the blank space to `<br>`. The format defined in OpenAPI documents is preserved in slate

**Comment by MikeRalphson** [comment]
Could you try amending the templates (either by editing them directly or setting the `-user_templates` option) to change the `join:true` option to `join:false` to see if that has the kind of effect you're looking for?

```
main.dot:{{ var blocks = data.utils.schemaToArray(origSchema,-1,{trim:true,join:true},data); }}
responses.def:{{ var blocks = data.utils.schemaToArray(responseSchema,0,{trim:true,join:true},data); }}
security.def:{{ for (var sc in flow.scopes) { }}|{{=sc}}|{{=data.utils.join(flow.scopes[sc])}}|
````

**Comment by revilwang** [comment]
I found the usage of `join: true` previously, but if I changed `true` to `false`, it will break the Markdown file, as the table cell is broken, shown in the following way:

```
|status|string|false|read-only|Order status

- `open`: waiting to be traded
- `finished`: finished|
```

So what I'm doing right now, is change the source code from `join: true` to `join: false` and add my own template render function to replace the blank space to `<br>`, but as I'm not going into the source code too deep, so I doubt I can submit a well-formed pull request. I leave this decision to you. 😆 

One more thing I don't get is how `security.def` is related to this.

**Comment by MikeRalphson** [comment]
> One more thing I don't get is how `security.def` is related to this.

It simply uses `data.join` to list the `scopes` of an `oath2` `securityScheme`.

**Comment by MikeRalphson** [comment]
Should be fixed in version `4.0.0`, which includes breaking changes (please see the `README.md` file).
